eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,24 @@
+parameters: 
+  TargetUserVariable: ''
+  TargetTeamVariable: 'notspecified'
+  TargetLabelVariable: 'notspecified'
+  ServiceDirectory: ''
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+variables:
+    - template: ../variables/tool-version.yml
+steps:
+  - pwsh: |
+      dotnet tool install --tool-path . --add-source "${{ parameters.DevOpsFeed }}" --version '$("Azure.Sdk.Tools.RetrieveCodeOwners")' "Azure.Sdk.Tools.RetrieveCodeOwners"
+    displayName: 'Install tool RetrieveCodeOwners'
+    workingDirectory: '$(Agent.ToolsDirectory)'
+
+  - pwsh: |
+      Push-Location '$(Agent.ToolsDirectory)'
+      .\retrieve-codeowners `
```

Comment:
```suggestion
      ./retrieve-codeowners `
```

Reply:
Also, what does this step do? Does the underlying tool output devops variables or a file or something?

Reply:
We fetch code owners from CODEOWNER file and put the owner into env targetUserVariable.
https://github.com/Azure/azure-sdk-for-python/blob/main/.github/CODEOWNERS#L13